docs/source/pages/using.rst

```diff
@@ -90,6 +90,50 @@ To analyze a repository on a self-hosted GitLab instance, you need to do the fol
 
 - Obtain a GitLab access token having at least the ``read_repository`` permission and store it into the ``MCN_SELF_HOSTED_GITLAB_TOKEN`` environment variable. For more detailed instructions, see `GitLab documentation <https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token>`_.
 
+''''''''''''''''''''''''''''''''''''''''''''''''''''
+Providing a PURL string instead of a repository path
+''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Instead of providing the repository path for the analysis software component, a `PURL <https://github.com/package-url/purl-spec/blob/master/PURL-SPECIFICATION.rst>`_. string
+for the target git repository could be used.
```

Comment:
```suggestion
Instead of providing the repository path to analyze a software component, you can use a `PURL <https://github.com/package-url/purl-spec/blob/master/PURL-SPECIFICATION.rst>`_. string for the target git repository.
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/401/commits/229e0394f00c7f257f46987c5549b0d72143aa2a